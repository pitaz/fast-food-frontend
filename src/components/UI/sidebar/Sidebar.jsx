import React, { Component } from "react";

class Sidebar extends Component {
  render() {
    return (
      <div id="mySidebar" className="sidebar">
        <a href="#">Admin</a>
        <a href="#">Menu</a>
        <a href="#">Ordered meals</a>
        <a href="#"> Logout</a>
      </div>
    );
  }
}

export default Sidebar;
