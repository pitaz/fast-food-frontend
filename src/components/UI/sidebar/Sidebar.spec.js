/* eslint-disable no-undef */
import React from 'react';
import expect from 'expect';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import Sidebar from './Sidebar.jsx';


describe('Sidebar Component', () => {
  const wrapper = shallow(<Sidebar />);
  it('should render the Sidebar component', () => {
    expect(wrapper.exists()).toBe(true);
  });
  it('should match the snapshot', () => {
    expect(toJson(wrapper)).toMatchSnapshot();
  });
});
