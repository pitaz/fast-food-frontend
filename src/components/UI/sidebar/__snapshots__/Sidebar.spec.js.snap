// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Sidebar Component should match the snapshot 1`] = `
<div
  className="sidebar"
  id="mySidebar"
>
  <a
    href="#"
  >
    Admin
  </a>
  <a
    href="#"
  >
    Menu
  </a>
  <a
    href="#"
  >
    Ordered meals
  </a>
  <a
    href="#"
  >
     Logout
  </a>
</div>
`;
